dist/js/medium-editor.js

```diff
@@ -1020,6 +1020,11 @@ MediumEditor.extensions = {};
             }
         },
 
+        /*
```

Comment:
Yeah not really because we still need them to push a new release.
That's why we put a text in the [PR template](https://github.com/yabwe/medium-editor/blob/master/.github/PULL_REQUEST_TEMPLATE.md#please-dont-submit-dist-files-with-your-pr) (which no one read) saying: 

**Please, don't submit `/dist` files with your PR!**